paddle/fluid/framework/block_desc.cc

```diff
@@ -238,5 +238,37 @@ BlockDesc *BlockDesc::ForwardBlock() const {
   return prog_->MutableBlock(static_cast<size_t>(desc_->forward_block_idx()));
 }
 
+void BlockDesc::MoveFrom(BlockDesc *block) {
+  PADDLE_ENFORCE_NOT_NULL(
+      block, platform::errors::InvalidArgument("block must be provided"));
```

Comment:
we recommend the first letter of error message sentence capitalized and ends with `.`, other cases are same

Reply:
Done.